drivebrain_core_base/include/public/ControllerManager.tpp

```diff
@@ -0,0 +1,119 @@
+
+#include <ControllerManager.hpp>
+
+template <typename ControllerType, size_t NumControllers>
+bool control::ControllerManager<ControllerType, NumControllers>::init()
+{
+    auto max_switch_speed = get_parameter_value<float>("max_controller_switch_speed_ms");
+    auto max_torque_switch = get_parameter_value<float>("max_torque_switch_nm");
+    auto max_accel_switch_request = get_parameter_value<float>("max_accel_switch_float");
+
+    if (!(max_switch_speed && max_torque_switch && max_accel_switch_request))
+    {
+        std::cout << "ERROR: couldnt get params" << std::endl;
+        return false;
+    }
+
+    if ((*max_accel_switch_request) > 1.0)
+    {
+        std::cout << "ERROR: max accel switch float is greater than max value possible (1.0)" << std::endl;
```

Comment:
Do we want an early return here so we don't keep this wrong accel_request value?